### ISSUE TITLE
Refreshing the page in edit mode crashes the app after changing the asset's price

### ISSUE THREAD

**Issue by EnzoVezzaro** [oceanprotocol/market#1379]
**Describe the bug**
If you reload the page when on editing, the app crashes.

**To Reproduce**
1. Go to Owned Asset
2. Click on Edit
3. Change the price and update
4. Re-open edit
5. Refresh page
6. See error

**Screenshots**
Video:
https://user-images.githubusercontent.com/13741335/166217519-56cb6ff5-7dd6-4716-be50-4cf96c0301c0.mp4

Live V4:
<img width="1440" alt="Screen Shot 2022-05-02 at 5 54 24 AM" src="https://user-images.githubusercontent.com/13741335/166217480-066a490c-41c5-474b-9760-06cd8324e7c8.png">

Local:
<img width="1440" alt="Screen Shot 2022-05-02 at 5 54 07 AM" src="https://user-images.githubusercontent.com/13741335/166217565-f8a25b6d-0631-4df7-86c7-c89700fe6336.png">

**Comment by jamiehewitt15** [comment]
Interesting, I didn't manage to recreate this. It didn't crash for me when I reloaded on the edit page

**Comment by EnzoVezzaro** [comment]
> Interesting, I didn't manage to recreate this. It didn't crash for me when I reloaded on the edit page

Interesting, are you sure you're testing [v4](https://v4.market.oceanprotocol.com/)? or an asset with an editable price field?

**Comment by mihaisc** [comment]
Can't replicate this as well.

**Comment by jamiehewitt15** [comment]
> Interesting, are you sure you're testing [v4](https://v4.market.oceanprotocol.com/)? or an asset with an editable price field?

Yeah, definitely testing on the V4 market

**Comment by EnzoVezzaro** [comment]
this happens consistently to me on any file with "Price" field.

https://user-images.githubusercontent.com/13741335/166229409-16c17ca2-a01f-4f98-8972-8ad4718cdc3f.mp4

**Comment by jamiehewitt15** [comment]
Very odd. I've tried clicking refresh several times but haven't been able to replicate the crash

https://user-images.githubusercontent.com/20739535/166230195-e1749c95-ca13-4c9f-a262-5968e8f51c62.mp4

**Comment by EnzoVezzaro** [comment]
> Very odd. I've tried clicking refresh several times but haven't been able to replicate the crash
> 
>  0x9c4e.9e11.-.Ocean.Market.-.2.May.2022.mp4

Yeah, very interesting. The first asset in your video should've crashed the app, the second is fine because it's not a fixed priced asset.

Another thing I did on this assets was changing their prices. Can try to edit the price and then recheck?

**Comment by EnzoVezzaro** [comment]
Now I'm not sure it's related to the price changing or there's something on my account. But see here in the video: I just published a new asset, after I change its price the refresh crashes the app. 

https://user-images.githubusercontent.com/13741335/166232733-46d6f52f-9e65-4933-bb8b-bb04ae767632.mp4

**Comment by jamiehewitt15** [comment]
Still a bit of a mystery. I've tried changing the price again and refreshing it a few more times but have not managed to make it crash.
 
https://user-images.githubusercontent.com/20739535/166234395-3cf70e1d-8948-47e5-889d-c485cb6a9643.mp4

**Comment by EnzoVezzaro** [comment]
> Still a bit of a mystery. I've tried changing the price again and refreshing it a few more times but have not managed to make it crash.
> 
>  0x9c4e.9e11.-.Ocean.Market.-.2.May.2022.1.mp4

Very very strange. We're both testing the v4 branch. I just published another asset again and encounter the same error. On my end it's pretty consistent, it keeps happening. More investigation is needed.

What I just did was:

- Created a new Fixed Price Asset
- Click on Edit
- Test refresh: here it works without problems
- Click on edit again
- Change the price
- Reopen edit again
- test refresh: the app crashes like in the video

This is happening on this newly created [asset](https://v4.market.oceanprotocol.com/asset/did:op:462ff6f02e0dec7853dacf1833189f2d1c18a72098c1afa2cd57028224107ee3)
